packages/codefixes/package.json

```diff
@@ -35,6 +35,7 @@
     "@rehearsal/utils": "workspace:*"
   },
   "devDependencies": {
+    "@rehearsal/upgrade": "workspace:*",
```

Comment:
isn't this a circular dependency?

Reply:
Maybe I'm missing something but it doesn't seemed used.

Reply:
:(

Reply:
Moved test back to `upgrade`... will update test later after CodeFixAction